File: 2. Arrays/23. Noble Integer/sol1.cpp
int Solution::solve(vector<int>  &A){
    sort(A.begin(), A.end());
    int size=A.size();
    if (A[size-1]==0)
        return 1;
    for (int j=0;j<(size-1);j++)
    {
        if (A[j]==A[j+1])
            continue;
        if (A[j]==size-1-j)
            return 1;
    }

    return -1;
}

File: 10. Backtracking/Subsets/sol1.py
class Solution:
    # @param A : list of integers
    # @return a list of list of integers
    def subsets(self, s):
        s.sort()
        r = [[]]
        for e in s:
            r += [x+[e] for x in r]
            print (r)
        return sorted(r)
sol_obj=Solution()
input=[1,2,3,4]
print (sol_obj.subsets(input))
